.azure-pipelines-canary.yml

```diff
@@ -27,18 +27,19 @@ stages:
   condition: and(succeeded(), eq(variables['IsCanary'], 'true'))
   dependsOn: Build_Canary
 
-  jobs:
-  - template: build/stage-release-appcenter.yml
-    parameters:
-      applicationEnvironment: Staging
-      deploymentEnvironment: AppCenter
-      appCenterWindowsSlug: $(AppCenterWindowsSlug_Canary)
-      appCenteriOSSlug: $(AppCenteriOSSlug_Canary)
-      appCenterAndroidSlug: $(AppCenterAndroidSlug_Canary)
-      androidKeyStoreFile: $(InternalKeystore)
-      androidVariableGroup: 'ApplicationTemplate.Distribution.Internal.Android'
-      appCenterServiceConnectionName: $(AppCenterCanaryServiceConnection)
-      appCenterDistributionGroup: $(AppCenterCanaryDistributionGroup)
+  # TODO create firebase project for the canary app
```

Comment:
Do we have something in the backlog for that? No need to add a link here but to be sure. @jeanplevesque

Reply:
I'm going to resolve the comment since I want to complete the PR and I don't think it will require a fix in the code